lib/core/network/api/aws_amplify/aws_authenticated_request.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:amazon_cognito_identity_dart_2/cognito.dart';
+import 'package:amazon_cognito_identity_dart_2/sig_v4.dart';
+import 'package:dio/dio.dart';
+import 'package:hypha_wallet/core/network/networking_manager.dart';
+
+///
+/// Send an AWS authenticated request
+///
+Future<dynamic> awsAuthenticatedRequest({
+  required CognitoCredentials credentials,
+  required String awsRegion,
+  String method = 'POST',
```

Comment:
this seems hacky 🤷

Reply:
Open for better ideas... but what else would I put there?

I want to be able to make GET requests too.